### ISSUE TITLE
Event Handling

### ISSUE THREAD

**Issue by DJ45X** [DKRCommands/DKRCommands#2]
In regards to [this](https://github.com/AlexzanderFlores/WOKCommands/issues/231#issuecomment-1276770390),

I'd rather not have my events in the index.js. Is there a way, through DKRCommands, to move my event code into a separate file/ folder? I liked how WOKCommands used to have a Features directory where I could store my event listeners. Helped a lot with file/ folder organization and keeping the index.js lean. 

Else, I'm open to any ideas you may have :)

**Comment by karelkryda** [comment]
Hello,
I have currently published version `1.0.0-beta.3`, which allows to load events from a separate folder with the following hierarchy:
```
> events
  > messageCreate
    > eventFile1.ts/.js
    > eventFile2.ts/.js
    > ...
  > messageDelete
    > eventFile1.ts/.js
    > eventFile2.ts/.js
    > ...
```
`events` = events folder
`messageCreate/Delete` = event folder - must have correct [name](https://discord.js.org/#/docs/main/main/class/Client?scrollTo=e-applicationCommandPermissionsUpdate)
each folder can have any number of files (when a given event is called, all of them are triggered)

\
I'll also update the documentation to reflect these changes in a few minutes.
I'll ask you to test and report any problems 🙂.

Have a nice Friday afternoon

\
*P.S. DKRCommands now use DJS v14.7.1 😉*

**Comment by karelkryda** [comment]
📖 Documentation updated: https://karel-kryda.gitbook.io/dkrcommands/useful-information/discord.js-events

**Comment by DJ45X** [comment]
> Hello, I have currently published version `1.0.0-beta.3`, which allows to load events from a separate folder with the following hierarchy:
> 
> ```
> > events
>   > messageCreate
>     > eventFile1.ts/.js
>     > eventFile2.ts/.js
>     > ...
>   > messageDelete
>     > eventFile1.ts/.js
>     > eventFile2.ts/.js
>     > ...
> ```
> 
> `events` = events folder `messageCreate/Delete` = event folder - must have correct [name](https://discord.js.org/#/docs/main/main/class/Client?scrollTo=e-applicationCommandPermissionsUpdate) each folder can have any number of files (when a given event is called, all of them are triggered)
> 
> I'll also update the documentation to reflect these changes in a few minutes. I'll ask you to test and report any problems 🙂.
> 
> Have a nice Friday afternoon
> 
> _P.S. DKRCommands now use DJS v14.7.1 😉_

Absolutely amazing! This is exactly what I needed! Thank you so much! 💜
